### DISCUSSION TITLE
quartz定时任务在海豚调度器重启后，在哪里重新上线执行？

### DISCUSSION THREAD

**Discussion by roamerop** [apache/dolphinscheduler#6365] [Q&A]
我启动了一个定时任务并上线运行，我想知道在海豚调度器重启后，源码中quartz是在哪里初始化把mysql中的定时任务开启的。（在源码中没找到，期待大神解答）

**Comment by davidzollo** [comment]
Usually I think you don't need to consider this question
the cron status was stored into database, you also can check the workflow status on the UI. the cron will work when the server starts
